uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/AbsenceTypeTest.php

```diff
@@ -601,4 +601,29 @@ private function createReservedType() {
 
     return null;
   }
+
+  public function testAbsenceTypeHasIsSickFlagAsFalseByDefault() {
+    $params = [
+      'title'                               => 'Title 1',
+      'color'                               => '#000101',
+      'default_entitlement'                 => 5,
+      'is_active'                           => 1,
+      'is_default'                          => 1
+    ];
+    $absenceType = AbsenceTypeFabricator::fabricate($params);
+    $this->assertEquals(0, $absenceType->is_sick);
+  }
+
+  public function testSetIsSickFlagForAbsenceType() {
+    $params = [
+      'title'                               => 'Title 1',
+      'color'                               => '#000101',
+      'default_entitlement'                 => 5,
+      'is_active'                           => 1,
+      'is_default'                          => 1,
+      'is_sick'                             => 1
+    ];
+    $absenceType = AbsenceTypeFabricator::fabricate($params);
```

Comment:
On those 2 tests, can you please use the create method instead of the fabricator? Since you're testing the creation behavior, it it's better to make that explicit.